optuna/visualization/_timeline.py

```diff
@@ -107,28 +116,29 @@ def _get_timeline_info(study: Study) -> _TimelineInfo:
 
 def _get_timeline_plot(info: _TimelineInfo) -> "go.Figure":
     _cm = {
-        "COMPLETE": "blue",
+        "COMPLETE": "#cccccc",
```

Comment:
I think feasible trials should be blue while infeasible trials will be `#cccccc`.

>  We discussed this among the committers and reached the conclusion that graying out should be applied to trials that are both infeasible and completed, as it might be clearer.